Dockerfile.java-alpine

```diff
@@ -77,12 +77,30 @@ RUN mkdir /app && \
     chown -R 10001:0 /app/volumes && \
     find /app \( -type d -exec chmod ug+rwx {} \; -exec chown 10001:0 {} \; -o -type f -exec chmod ug+rw {} \; -exec chown 10001:0 {} \; \)
 
+ENV JAVA_OPTIONS=
+ENV QUBERSHIP_PROFILER_ENABLE=N
+ARG QUBERSHIP_PROFILER_VERSION=1.0.4
```

Comment:
Renovate can automatically update the profiler version, so consider using Renovate